packages/cozy-harvest-lib/src/components/KonnectorConfiguration/ConfigurationTab/BIContractActivationWindow.spec.jsx

```diff
@@ -86,6 +87,7 @@ describe('BIContractActivationWindow', () => {
     )
   })
   it('should call InAppBrowser display if in flagship app context', async () => {
+    expect.assertions(7)
```

Comment:
to be removed?

Reply:
No. Jest considers the test as success once at least an expectation is in success. However inside this long test, there is long tasks (waitFor) that are somehow skipped, because Jest continues testing other file. But the waitFor is still launched, outside its own test, and that provokes a warning (from jest-dom) making the whoe test suite failing.